charts/kyverno/templates/_helpers.tpl

```diff
@@ -48,7 +48,7 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 app.kubernetes.io/name: {{ template "kyverno.name" . }}-test
 app.kubernetes.io/part-of: {{ template "kyverno.name" . }}
-app.kubernetes.io/version: "{{ .Chart.Version }}"
+app.kubernetes.io/version: "{{ .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}"
```

Comment:
Do we need this part `| trunc 63 | trimSuffix "-"` ?

Reply:
I guess not - let me remove it

Reply:
@eddycharly removed

Reply:
Thanks